Core/Security/ProxyInfo.py

```diff
@@ -168,7 +166,7 @@ def formatProxyStepsInfoAsString(infoList):
         if key == "extensions":
           value = "\n   %s" % "\n   ".join(["%s = %s" % (extName.strip().rjust(20), extValue.strip())
                                             for extName, extValue in value])
-        contentsList.append("  %s : %s" % (key.ljust(10).capitalize(), value))
+        contentsList.append("  %s : %s" % (key.ljust(10).capitalize(), str(value)))
```

Comment:
The `%s` inside the quote is already doing the `str` cast

Reply:
fixed